### DISCUSSION TITLE
PyPy released a new version, with support for python3.9, 3.10

### DISCUSSION THREAD

**Discussion by mattip** [pypa/cibuildwheel#1524] [General]
It would be nice if cibuildwheel supported the newer PyPy versions in its next release.

Also: issue #1400 to support cross-compilation for PyPy macos arm64 did not get much attention. What would be the steps to enable this?

**Comment by henryiii** [comment]
Needs https://github.com/pypa/manylinux/pull/1500

**Reply by mayeut** [reply]
I opened #1525 now that https://github.com/pypa/manylinux/pull/1500 has been merged.

**Comment by mattip** [comment]
cool, thanks
